examples/logical_enclosures.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.exceptions import HPOneViewException
+from config_loader import try_load_from_file
+
+config = {
+    "ip": "172.16.102.59",
+    "credentials": {
+        "userName": "administrator",
+        "password": ""
+    }
+}
+
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# Get first logical enclosure
+logical_enclosure = oneview_client.logical_enclosures.get_all()[0]
+print("Found logical enclosure '{}' at\n   uri: '{}'".format(
+    logical_enclosure['name'], logical_enclosure['uri']))
+
+# Update the logical enclosure name
+logical_enclosure_name = logical_enclosure['name']
+logical_enclosure['name'] = logical_enclosure_name + "-updated"
+print("Update the logical enclosure to have a name of '%s'" %
+      logical_enclosure['name'])
+logical_enclosure = oneview_client.logical_enclosures.update(logical_enclosure)
+print("   Done. uri: '%s', 'name': '%s'" %
+      (logical_enclosure['uri'], logical_enclosure['name']))
+print("Reset name")
+logical_enclosure['name'] = logical_enclosure_name
+logical_enclosure = oneview_client.logical_enclosures.update(logical_enclosure)
+print("   Done.")
```

Comment:
Since it's a logical enclosure that was already on the system (they can't be created or deleted for c7000s), I didn't want to leave it with an '-updated' name. If I did, then if the example was run more than once, there would be something with the name '-updated-updated' and so on.